books/RayTracingInOneWeekend.html

```diff
@@ -90,7 +90,7 @@
 
         std::cout << "P3\n" << image_width << ' ' << image_height << "\n255\n";
 
-        for (int j = image_height-1; j >= 0; --j) {
```

Comment:
Why the change?

I don't have a preference either way, but I'm curious the motivation

Reply:
Especially since it's flipping the first rendered texture

Reply:
Pixels have coordinates in 3D space which trivially mirror to the XY axis in world 3-space, but that only applies for the initial camera position & orientation and the placement of the image plane. As always, there's conflict with the Y axis. Generally, I prefer to keep spaces consistent instead of having to remember what my context is in the different space.

In this case, I want the following: the `scene` class is responsible for interrogating pixels, based on their image coordinates (which are Y down). The `camera` class is positioned and oriented in 3-space, but uses image coordinate conventions when interrogating specific pixels. Thus the `camera.get_ray()` method takes in pixel coordinates, which are now consistently in image coordinates.

The flipped rendered texture is no big deal, since there's no correct orientation. I plan to re-render this image during the v4.0.0 prep phase.